src/Analyzers/CSharp/Analyzers/CSharpAnalyzersResources.resx

```diff
@@ -289,4 +289,7 @@
   <data name="Suppression_operator_has_no_effect_and_can_be_misinterpreted" xml:space="preserve">
     <value>Suppression operator has no effect and can be misinterpreted</value>
   </data>
+  <data name="Simplify_linq_expressions" xml:space="preserve">
+    <value>Simplify linq expressions</value>
```

Comment:
Linq is a proper name, so should likely be first-letter-capitalized.    I would also expect thsi string to be `Simplify Linq expression` (singular).  but maybe the code will make it clear why it is plural.